release/release_build.py

```diff
@@ -245,44 +250,37 @@ def main(argv):
     # SEMIAUTO Indicates a mostly automated step with possible prompts. Most of these steps become fully-automated when --auto is used.
     # AUTO Indicates the step is fully-automated.
 
-    # Note that many prompts will cause scripts to exit if you say 'no'. This will require you to re-run
-    # the script from the beginning, which may take a long time. It is better to say 'yes' to the script
-    # prompts and follow the indicated steps even if they are redundant/you have done them already. Also,
-    # be sure to carefully read all instructions on the command-line before typing yes. This is because
-    # the scripts do not ask you to say 'yes' after each step, so you may miss a step if you only read
-    # the paragraph asking you to say 'yes'.
+    delete_if_exists( RELEASE_BUILD_COMPLETED_FLAG_FILE )
 
     set_umask()
 
     projects_to_release = read_projects( argv, print_usage )
+
+    #Check for a --auto
```

Comment:
I suggest inserting a space between the comment syntax "#" and the following text.